node-fs-readfile/cat.js

```diff
@@ -0,0 +1,17 @@
+const fs = require('fs');
+let dataStorage = '';
+let boolean = false;
+
+for (let i = 2; i < process.argv.length; i++) {
+  const fileName = process.argv[i];
+  fs.readFile(fileName, 'utf8', (err, data) => {
+    if (err) throw err;
+    dataStorage = dataStorage + data + '\n';
+    if (i === process.argv.length - 1) {
+      boolean = true;
+    }
+    if (boolean === true) {
+      console.log(dataStorage);
+    }
+  });
+}
```

Comment:
for the record, this level 3 solution doesn't guarantee the print order of the file contents.